### PR TITLE
Modified about.html files

### PR DESCRIPTION
**Pull request by kirilAtan**
Eclipse foundation changed to openHab community

Signed-off-by: Kiril Atanasov <kiril.atanasov@musala.com>

### CODE REVIEW

**Comment by kirilAtan** [comment]
Files, which were not modified:
openhab2-addons/addons/voice/org.openhab.voice.marytts/about.html
openhab2-addons/addons/voice/org.openhab.voice.kaldi/about.html
openhab2-addons/addons/io/org.openhab.io.homekit/about.html
All modified files had date, licence agreement information and some of them had information about third-party licences. These 3 files had only information about third-party licences.

File: addons/binding/org.openhab.binding.feed/aboutFilesModifications.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+about.html files modified
```
Comment (kirilAtan) [inline addons/binding/org.openhab.binding.feed/aboutFilesModifications.txt]: I am terribly sorry for that.

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.feed/aboutFilesModifications.txt]: No worries, thanks for the PR!

**Comment by kirilAtan** [comment]
I have a proposition to add example about.html files in the coding guidelines.
https://github.com/openhab/openhab-docs/pull/109

**Pull request merged**